src/materials/minetest.lua

```diff
@@ -17,13 +18,17 @@ local materials = {
     axe_stone = "default:axe_stone",
     axe_wood = "default:axe_wood",
     pick_steel = "default:pick_steel",
+    pick_silver = "default:pick_silver",
```

Comment:
```suggestion
    pick_silver = "moreores:pick_silver",
```

Reply:
this is a draft, not ready for review

Reply:
i know, but i assumed it would be useful to get some feedback now which might help you to not forget when you work on this next.